javagen/src/main/java/com/azure/autorest/template/ClientMethodTemplate.java

```diff
@@ -762,79 +777,53 @@ private static String parameterDescriptionOrDefault(ProxyMethodParameter paramet
+                            function.line("res.getValue().%s(),", CodeNamer.getModelNamer().modelPropertyGetterName(clientMethod.getMethodPageDetails().getNextLinkName()));
                         }
-                    });
-                });
-            });
-        } else {
-            writeMethod(typeBlock, clientMethod.getMethodVisibility(), clientMethod.getDeclaration(), function -> {
-                AddValidations(function, clientMethod.getRequiredNullableParameterExpressions(), clientMethod.getValidateExpressions(), settings);
-                AddOptionalAndConstantVariables(function, clientMethod, restAPIMethod.getParameters(), settings);
-                ApplyParameterTransformations(function, clientMethod, settings);
-                ConvertClientTypesToWireTypes(function, clientMethod, restAPIMethod.getParameters(), clientMethod.getClientReference(), settings);
-
-                String serviceMethodCall = checkAndReplaceParamNameCollision(clientMethod, restAPIMethod, settings);
-                if (settings.getAddContextParameter()) {
-                    if (settings.isContextClientMethodParameter() && contextInParameters(clientMethod)) {
-                        function.line(String.format("return %s", serviceMethodCall));
                     } else {
-                        function.line(String.format("return FluxUtil.withContext(context -> %s)",
-                            serviceMethodCall));
-                    }
-                } else {
-                    function.line(String.format("return %s",
-                            serviceMethodCall));
-                }
-                function.indent(() -> {
-                    function.line(".map(res -> new PagedResponseBase<>(");
-                    function.indent(() -> {
-                        function.line("res.getRequest(),");
-                        function.line("res.getStatusCode(),");
-                        function.line("res.getHeaders(),");
-                        function.line("res.getValue().%s(),", CodeNamer.getModelNamer().modelPropertyGetterName(clientMethod.getMethodPageDetails().getItemName()));
                         function.line("null,");
-                        IType responseType = ((GenericType) clientMethod.getProxyMethod().getReturnType()).getTypeArguments()[0];
-                        if (responseType instanceof ClassType) {
-                            function.line("res.getDeserializedHeaders()));");
-                        } else {
-                            function.line("null));");
-                        }
-                    });
+                    }
+                    IType responseType = ((GenericType) clientMethod.getProxyMethod().getReturnType()).getTypeArguments()[0];
+                    if (responseType instanceof ClassType) {
+                        function.line("res.getDeserializedHeaders()));");
+                    } else {
+                        function.line("null));");
+                    }
```

Comment:
What is changed here? It seems removed an `if` branch.

Reply:
Yes. OId `if` and `else` have huge code clone. I also add code for low level client.